.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Please remove this file from commit and add this in ignore list.

Reply:
added

Reply:
added in ignore list